README.rst

```diff
@@ -23,8 +23,6 @@ Disclosure
 -  This code has not been through a third party security audit.
 -  I’ve successfully tested this locally with ``pypy-5.4.1``. TravisCI
    has confirmed this doesn’t work with their version.
--  From their logs:
-   ``RuntimeError: PyNaCl is not compatible with PyPy < 2.6. Please upgrade PyPy to use this library.``
 
```

Comment:
Won't that confuse users?  DCN never "supported" PyNacl, it was a dependency, and now it's not a dependency.   DCN itself is still backwards compatible because the interface hasn't changed.  If there was a ChangeLog file, the PyNacl removal could go in there.

If anything, maybe consider adding something about attempting to only use "pure Python" dependencies for best compatibility with PyPy and AppEngine?